### PR TITLE
Austinscreations new pages

### PR DESCRIPTION
**Pull request by austinscreations**
adds new topics and title pages for more information to be added
- added page room8266 in docs -> hardware -> controllers
- added I2CRJ45 in docs -> hardware -> input
- added PWM Controller in docs -> hardware - output
- added category "Cases" in docs
- added title page to cases in docs-> cases
- added sub page for rack mount cases in docs-> cases
- added sub page for 3D printed cases in docs-> cases
- added heading to get started guide in guides
- moved advanced guide to folder in guides
- added title page to advanced folder in guides -> advanced
- added standards folder in guides
- added IDC guide in guides -> standards
- added Hardware / software versioning in guides -> standards
- added Ethernet IO in guides -> standards

### CODE REVIEW

File: docs/docs/hardware/controllers/room8266.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## Features
+
+- **ESP8266** microcontroller with **WiFi**.
+- **10/100Mbps** Ethernet.
+- Support for **802.3af PoE**.
+- 2.1mm Power jack for optional external DC power input.
+- 6-way IDC "I2C-Breakout" header for linking to other boards.
+- 2x 4 pin jst sh conenctors to match teh Stemma / qwiic connector i2c standard. 
+- On baord RGBW sk6812 LED.
```
Comment (sumnerboy12) [inline docs/docs/hardware/controllers/room8266.md:25]: baord

File: docs/docs/cases/3D-printed/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+tags: [""]
+---
+# 3D Printed Cases
+
+## Overview
+Sometimes you don't really need a fully rack to run a proejct and a simple case that lets you palce an item nicely in a local spot is better suited.
```
Comment (sumnerboy12) [inline docs/docs/cases/3D-printed/README.md:7]: fully -> full

Comment (sumnerboy12) [reply docs/docs/cases/3D-printed/README.md:7]: Typo `palce`

File: docs/docs/cases/3D-printed/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+A great way to do this is by using a 3D printed Case
```
Comment (sumnerboy12) [inline docs/docs/cases/3D-printed/README.md:8]: Case -> case.

File: docs/docs/hardware/controllers/room8266.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+# Room8266
+<p class="maker">by <b>Austin's Creations</b></p>
+
+> SKU: OXRS-AC-ROOM8266
+
+<!-- Board Image -->
+![Austin's Creations Room8266 ESP8266 control board](/images/oxrs-room8266.jpg)
+
+<!-- Board Description -->
+A general purpose control board for simple i2c local baord control mount projects.
```
Comment (sumnerboy12) [inline docs/docs/hardware/controllers/room8266.md:13]: baord

File: docs/docs/hardware/controllers/room8266.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+- **10/100Mbps** Ethernet.
+- Support for **802.3af PoE**.
+- 2.1mm Power jack for optional external DC power input.
+- 6-way IDC "I2C-Breakout" header for linking to other boards.
+- 2x 4 pin jst sh conenctors to match teh Stemma / qwiic connector i2c standard. 
+- On baord RGBW sk6812 LED.
+
+## Supported Firmware
+- OXRS-AC-LedController-ESP8266-FW [Github](https://github.com/austinscreations/OXRS-AC-LedController-ESP8266-FW)
+- OXRS-AC-LedController-ESP32-FW  [Github](https://github.com/austinscreations/OXRS-AC-LedController-ESP32-FW)
```
Comment (sumnerboy12) [inline docs/docs/hardware/controllers/room8266.md:29]: Not sure the ESP32 version will work on this board?

File: docs/docs/hardware/controllers/room8266.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+## Supported Libraries
+- OXRS-SHA-Rack32-ESP32-LIB [Github](https://github.com/SuperHouse/OXRS-SHA-Rack32-ESP32-LIB)
```
Comment (sumnerboy12) [inline docs/docs/hardware/controllers/room8266.md:32]: Don't think the Rack32 library should be linked here?

File: docs/docs/hardware/input-devices/I2CRJ45.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+# I2CRJ45
+<p class="maker">by <b>SuperHouse Automation</b></p>
+
+<!-- Board Image -->
+![SuperHouse Automation I2CRJ45x8 board](/images/oxrs-I2CRJ45X8.jpg)
+![SuperHouse Automation I2CRJ45x16 board](/images/oxrs-I2CRJ45X16.jpg)
+![SuperHouse Automation I2CRJ45x24 board](/images/oxrs-I2CRJ45X24.jpg)
+
+<!-- Board Description -->
+I2C compactible baord that handles multiple inputs via an ethernet RJ45 connection
```
Comment (sumnerboy12) [inline docs/docs/hardware/input-devices/I2CRJ45.md:13]: compactible -> compatible
baord -> board
remove "ethernet" - it is just an RJ45 connection

File: docs/docs/hardware/input-devices/I2CRJ45.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+## Features
+- Coems in 8,16,24 port versiosn (32 port in the works)
```
Comment (sumnerboy12) [inline docs/docs/hardware/input-devices/I2CRJ45.md:16]: Coems -> Comes

File: docs/docs/hardware/input-devices/I2CRJ45.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+- Firmware supports multiple types of switches, contacts, encoders
+- 12-24v DIY POE on teh conenctiosn for switches with basic LEDs
```
Comment (sumnerboy12) [inline docs/docs/hardware/input-devices/I2CRJ45.md:18]: teh conenctiosn -> then connections

File: docs/docs/hardware/output-devices/pwm-controller.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+
+## Supported Firmware
+- OXRS-AC-LedController-ESP8266-FW [Github](https://github.com/austinscreations/OXRS-AC-LedController-ESP8266-FW)
+- OXRS-AC-LedController-ESP32-FW  [Github](https://github.com/austinscreations/OXRS-AC-LedController-ESP32-FW)
+
+## Additional Resources
+- Additional Resource 1 [GITHUB](https://github.com/austinscreations/PWM-LED-Controller)
+
+## Where to Buy
+- Conatact on Discord or website [WEBSITE](https://www.austinscreations.ca/)
```
Comment (sumnerboy12) [inline docs/docs/hardware/output-devices/pwm-controller.md:28]: Conatact -> Contact

File: docs/guides/getting-started.md [hunk 11]
```diff
@@ -2,7 +2,26 @@
 tags: [""]
 ---
 # Getting Started Guide
-
+ A great adventure awaits those using the OXRS platform, and with teh help of this Guide you'll be able to pick you adventure.
```
Comment (sumnerboy12) [inline docs/guides/getting-started.md:5]: teh -> the

**Review by sumnerboy12 — COMMENTED** [review]
Great stuff Austin - noticed a few typos in there, nothing major tho. Tidy those up and I will merge this when you are happy it is ready.

**Review by austinscreations — COMMENTED** [review]
made the changes, hopefully it should be good to go.
i actually noticed a few spelling things so fix them as well.

**Pull request merged**